cluster/mongodb-atlas-cluster.json

```diff
@@ -0,0 +1,289 @@
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        },
+        "RegionsConfig": {
+            "type": "object",
+            "properties": {
+                "RegionName": {
+                    "type": "string"
+                },
+                "AnalyticsNodes": {
+                    "type": "integer"
+                },
+                "ElectableNodes": {
+                    "type": "integer"
+                },
+                "Priority": {
+                    "type": "integer"
+                },
+                "ReadOnlyNodes": {
+                    "type": "integer"
+                }
+            }
+        },
+        "ReplicationSpec": {
+            "type": "object",
+            "properties": {
+                "ID": {
+                    "type": "string"
+                },
+                "NumShards": {
+                    "type": "integer"
+                },
+                "RegionsConfig": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/RegionsConfig"
+                    }
+                },
+                "ZoneName": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        },
+        "apiKeyDefinition": {
+            "type": "object",
+            "properties": {
+                "PublicKey": {
+                    "type": "string"
+                },
+                "PrivateKey": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        }
+    },
+    "properties": {
+        "ApiKeys": {
+            "$ref": "#/definitions/apiKeyDefinition"
+        },
+        "AutoScaling": {
+            "type": "object",
+            "properties": {
+                "Compute": {
+                    "type": "object",
+                    "properties": {
+                        "Enabled": {
+                            "type": "boolean"
+                        },
+                        "ScaleDownEnabled": {
+                            "type": "boolean"
+                        }
+                    }
+                },
+                "DiskGBEnabled": {
+                    "type": "boolean"
+                }
+            },
+            "additionalProperties": false
+        },
+        "BackupEnabled": {
+            "description": "Applicable only for M10+ clusters. Set to true to enable Atlas continuous backups for the cluster. Set to false to disable continuous backups for the cluster. Atlas deletes any stored snapshots. See the continuous backup Snapshot Schedule for more information. You cannot enable continuous backups if you have an existing cluster in the project with Cloud Provider Snapshots enabled. The default value is false.",
+            "type": "boolean"
+        },
+        "BiConenctor": {
```

Comment:
Is this an errata? if so, it should be `Biconnector`, isn't it?

Reply:
You are right! I'll do the change.